tests/mock_tables/global_db/appl_db.json

```diff
@@ -20,6 +20,6 @@
     "lldp_loc_sys_desc": "Gotta go Fast!",
     "lldp_loc_sys_cap_enabled": "28 00",
     "lldp_loc_sys_cap_supported": "28 00",
-    "lldp_loc_man_addr": "10.224.25.36,fe80::ce37:abff:feec:de9c"
+    "lldp_loc_man_addr": "fe80::ce37:abff:feec:de9c"
```

Comment:
The same table is present in other namespace APP_DBs also, can you make the same change in those as well, though it is not used in this MIB. https://github.com/Azure/sonic-snmpagent/blob/master/tests/mock_tables/asic0/appl_db.json#L30

Reply:
Resolved

---
In reply to: [505910755](https://github.com/Azure/sonic-snmpagent/pull/167#discussion_r505910755) [](ancestors = 505910755)